src/components/organisms/tab-bar-bottom.tsx

```diff
@@ -162,11 +156,20 @@ export const TabBarBottom: FC<any> = ({navigation, state}) => {
               accessibilityHint={hint}
               accessibilityState={{selected: isActive}}
               onPress={() => navigation.navigate(routeName)}>
-              <View style={[styles.tab, isActive ? styles.highlighted : {}]}>
+              <View
+                style={[
+                  styles.tab,
+                  isActive ? styles.highlighted : {},
```

Comment:
Isn't better some inline here rather than having highlighted & alertsActive (which over-ride the same props)

Reply:
inline styling? I'd rather not have that.

Reply:
Inline ifs, but as it is it's fine 👍